nixos/modules/services/networking/bee/bee-clef.nix

```diff
@@ -0,0 +1,203 @@
+{ config, lib, pkgs, ... }:
+
+# NOTE: for now nothing is installed into /etc/bee-clef/. The config
+# files are used as read-only from the nix store, because they are
+# not really variable. Open an issue if you need to change them.
+
+with lib;
+let
+  cfg = config.services.bee-clef;
+
+  deriveDataDir = id:
+    if id == null then "${cfg.dataDir}" else "${cfg.dataDir}/${toString id}";
+
+  makeSystemdTmpfilesEntry = id:
+    let
+      dataDir = deriveDataDir id;
+    in
+      [
+        "d '${dataDir}/'         0750 ${cfg.user} ${cfg.group}"
+        "d '${dataDir}/keystore' 0700 ${cfg.user} ${cfg.group}"
+      ];
+
+  makeSystemdServiceEntry = id:
+    let
+      dataDir        = deriveDataDir id;
+      namePostfix    = if id == null then "" else "-${toString id}";
+    in {
+      name = "bee-clef${namePostfix}";
+      value = {
+        path = [
+          # these are needed for the ensure-clef-account script
+          pkgs.coreutils
+          pkgs.gnused
+          pkgs.gawk
+        ];
+
+        wants    = [ "network.target" ];
+        wantedBy = [ "bee${namePostfix}.service" "multi-user.target" ];
+
+        serviceConfig = {
+          Nice = cfg.daemonNiceLevel;
+          NoNewPrivileges = true;
+          User = cfg.user;
+          Group = cfg.group;
+          ExecStartPre = ''${cfg.package}/share/bee-clef/ensure-clef-account'';
+          ExecStart = "${cfg.package}/share/bee-clef/bee-clef-service start";
+          ExecStop ="${cfg.package}/share/bee-clef/bee-clef-service stop";
+          Environment = [
+            #''"AUDIT_LOG_FILE=${cfg.auditLogDir}/clef-audit-log${namePostfix}.log"''
```

Comment:
Whats with this entry?

Reply:
i considered turning on audit logging by default, and installing a config for it, but the log file's size is not trivial, so i backed out. instead of deleting what i had, i just commented it out.

do you prefer it deleted instead?